server/meshmodel/policies/network.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Will policies wont be stored as rego files now?

If yes, then would it create readability and maintainability problem.

I think, they should be stored as .rego files but the script should take care to convert it to json, there could be a json -> rego file mapping, in case we cannot write metadata automatically on bootup.

Reply:
Yes, there should be a script that converts .rego -> .json file. Infact, I used a script to generate this `network.json`

The reason I removed .rego file from `/server/meshmode/policies` directory because it made things unsymmetrical considering both `/server/meshmode/relationships` and `/server/meshmode/components`directories has .json files.

I can add a `rego_to_json_script` under `/scripts` directory where we store other scripts for auto generation of MeshModel components from spreadsheet.

But then the questions would be
1. Where to store .rego files?
2. When those .rego files would be converted to .json files? It could be generated nightly just like we generate components....

// @leecalcote